kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/documentation/DocumentationLinksWidget.less

```
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[data-i18n-prefix="DocumentationLinksWidget."] {
+  .kie-documentation-links-container {
+    width: 100%;
+  }
+
+  //This is a workaround for positioning the 'manage' link on the same line as the label. The label is
+  //provided by a _consumer_ of this widget and consequentially we cannot interfere with the containing DIVs
+  //the _consumer_ uses for label and widget separation.
+  .kie-documentation-links-container #add-container {
+    position: absolute;
+    top: -24px;
+    right: 0px;
+  }
+
+  .documentation-links-container {
+    height: 40px;
+    overflow-y: auto;
+    overflow-x: hidden;
+  }
+
+  .col-md-10, col-md-2 {
```

I think we can remove this class:
```suggestion
  .col-md-10 {
```